### ISSUE TITLE
Styling for notes ought to be less subdued

### ISSUE THREAD

**Issue by rkirsling** [tc39/ecma262#1825]
# Problem

Other specs tend to style notes in a way that calls attention with color while retaining adequate subtlety, yet we style them as if they were ~being consumed by encroaching fog~ blockquotes in GitHub Markdown.

## WASM
![image](https://user-images.githubusercontent.com/2172590/71706170-0a576500-2d98-11ea-87f2-89d99daf2f8c.png)
## HTML
![image](https://user-images.githubusercontent.com/2172590/71706175-15aa9080-2d98-11ea-9050-5b34c97771b2.png)
## CSS
![image](https://user-images.githubusercontent.com/2172590/71706302-daf52800-2d98-11ea-801e-bd29b2561e38.png)
## ECMAScript
![image](https://user-images.githubusercontent.com/2172590/71706818-c8c8b900-2d9b-11ea-87d7-e48db8a05450.png)

But notes are meant to be read—they may be "inessential" from the view of the spec as an implementation in itself, but they are often of key importance to the human reader, and I question whether we really mean to visually "background" them as such. 

# Solution(s)

I think we can do better, with a bit of Ecma orange! Here are a few possibilities.

## 1. Higher-impact approach
![image](https://user-images.githubusercontent.com/2172590/71708953-4ba44080-2da9-11ea-9091-dacc989192ca.png)

Specifically:
- Make border full Ecma orange (like the [metadata box](https://tc39.es/ecma262/#metadata-block)), remove grey text color
- Add background color which is Ecma orange with lightness bumped to 95%
  (also make `<code>` background transparent and `<td>` background white)
- Add 10px padding on the other three sides

Notes:
- I say high-_er_ impact because we could go further still and restyle the "NOTE" indicator if we wanted.
- Since the border is full-colored, the background lightness here could even be bumped up further.

## 2. Medium-impact approach
![image](https://user-images.githubusercontent.com/2172590/71710058-37634200-2daf-11ea-85c6-f6c8fded23ce.png)

Like the first approach, but without a border:
- Remove grey border and its padding, remove grey text color
- Add background color which is Ecma orange with lightness bumped to 95%
  (also make `<code>` background transparent and `<td>` background white)
- Add 10px padding on all sides

Notes:
- This might be slightly less good for the printed version, where I believe background color would be omitted but border and padding would be preserved.

## 3. Lowest-impact approach
![image](https://user-images.githubusercontent.com/2172590/71709643-5234b700-2dad-11ea-9a40-fa22ca03c3ea.png)

Basically the complement of the second approach:
- Make border full Ecma orange (like the [metadata box](https://tc39.es/ecma262/#metadata-block))...and that's it.

Notes:
- I don't care for this since I really want to do something about the grey text color too, but if the note text is black-on-white just like the body text, I fear the border color alone isn't enough of a visual distinction.

-----

Please note that I'm raising this discussion here, but the PR would need to take place in the ecmarkup repo, assuming we don't want it to be restricted to ECMA-262.

**Comment by rkirsling** [comment]
Oh wow, I just realized that 402 is already using basically the exact style I've proposed here for a new element `<emu-normative-optional>`:

![image](https://user-images.githubusercontent.com/2172590/71772115-66d29580-2efa-11ea-94bf-0846e482c40a.png)

If Ecma orange is effectively doubling as a "warning" color there, then perhaps we should apply the same style to notes but use green or blue as the color instead?

**Comment by rkirsling** [comment]
So as a concrete proposal then, how about we do approach 1 above but with Bikeshed green?

I guess that still leaves a choice as to whether to color the "NOTE" indicator or not though.

With:
> ![image](https://user-images.githubusercontent.com/2172590/71789525-63b1d500-2fe0-11ea-814f-d8e50318f596.png)


Without:
> ![image](https://user-images.githubusercontent.com/2172590/71789544-7a582c00-2fe0-11ea-92dc-871080cf2aae.png)

Perhaps the former is more consistent?
